### DISCUSSION TITLE
Publishing game<title>

### DISCUSSION THREAD

**Discussion by Magnus-Wizard** [community/community#62447] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi please I just want to host my apk here so I can share the download link easily from my app but I don't know how to go about it and I don't know if it's even allowed. Please if I'm not supposed to ask this here don't delete this without first replying. Thanks
I already have the apk I just want to upload it and generate a link

**Comment by eru123** [comment]
If it's a public repository. You can publish a release, and along with the release drag and drop the apk file in binary files section. Here is an example repository with apk files https://github.com/tachiyomiorg/tachiyomi/releases

**Comment by h7x4** [comment]
I think the best, but not necessarily easiest way to go about this would probably be to automate builds using GitHub actions. It takes some effort to set up the first time, but then you have automated the build process for the future. There already exists a few building blocks to help you out with this:

- https://github.com/marketplace/actions/setup-android-sdk-tools
- https://github.com/marketplace/actions/sign-android-release
- https://github.com/marketplace/actions/upload-android-release-to-play-store
- https://github.com/marketplace/actions/automated-build-android-app-with-github-action
- https://github.com/actions/upload-artifact

There is also a nice [blogpost from logrocket](https://blog.logrocket.com/android-ci-cd-using-github-actions/) where they create the pipeline from scratch.

But if you just want to do a quick and dirty upload, then using a release like mentioned earlier is probably the easiest solution.

**Comment by queenofcorgis** [comment]
Thanks @h7x4 and @eru123 for the thoughtful replies and thank you for posting @Magnus-Wizard ❤️ This question is allowed, however posting the APK in the community would not be just for reference.

**Reply by h7x4** [reply]
> I just want to host my apk here

> posting the APK in the community would not be just

I could be wrong, but I think @Magnus-Wizard might have been referring to GitHub itself in the question? I'm a bit confused :sweat_smile:

**Reply by queenofcorgis** [reply]
@h7x4 - this was in reference to: 
>Please if I'm not supposed to ask this here don't delete this without first replying

But thank you for clarifying!
